staff/models.py

```diff
@@ -72,7 +72,7 @@ class Meta:
     ophase = models.ForeignKey(Ophase, models.CASCADE)
     prename = models.CharField(max_length=60, verbose_name=_('first name'))
     name = models.CharField(max_length=75, verbose_name=_('last name'))
-    email = models.EmailField(verbose_name=_("E-Mail-Adresse"))
+    email = models.EmailField(verbose_name=_("E-Mail-Adresse"), unique=True)
```

Comment:
Oh yeah, you are right. In fact, I was wondering why there was no constraint yet since I forgot about the unique_together constraint we introduced such a long time ago.
The obsolete constraint is removed again with 9e12145